include/wlr/xwayland.h

```diff
@@ -119,6 +119,13 @@ struct wlr_xwayland_surface_size_hints {
 	uint32_t win_gravity;
 };
 
+enum wlr_xwayland_icccm_input_state {
+	ICCCM_INPUT_STATE_NONE = 0,
```

Comment:
Please prefix each symbol with `WLR_`, to avoid potential conflicts with another library.

Reply:
No problem.